web-app/src/pages/tournament/TournamentsList.vue

```diff
@@ -46,50 +51,46 @@ export default class TournamentsList extends Vue {
       field: 'status',
       sortable: true,
     },
-    {
-      name: 'participants',
-      align: 'right',
-      label: 'Participants',
-      field: 'participants',
-    },
-  ];
-  private data = [
-    {
-      id: '1',
-      name: 'Work',
-      status: 'In progress',
-      participants: 'You and I',
-    },
-    {
-      id: '2',
-      name: 'Tourn2',
-      status: 'Finished',
-      participants: 'You and I',
-    },
-    {
-      id: '3',
-      name: 'Tourn3',
-      status: 'Finished',
-      participants: 'No one',
-    },
-    {
-      id: '4',
-      name: 'Tourn4',
-      status: 'Finished',
-      participants: 'No one',
-    },
-    {
-      id: '5',
-      name: 'Tourn5',
-      status: 'Finished',
-      participants: 'No one',
-    },
-    {
-      id: '6',
-      name: 'Tourn6',
-      status: 'Finished',
-      participants: 'No one',
-    },
   ];
+
+  @Watch('pagination.page')
+  private async pageChanged() {
+    await this.loadData(this.pagination.page);
+  }
+  private async loadData(page: number) {
+    //Funkcja wyciąga wszystkich uczestników danego turnieju.
+    const mapUsers = (teams: Team[]) => {
+      //1 etap - wynik: np. [[{firstName: "Artur", lastName: "Oswald"},{firstName: "Chris", lastName: "Green"}],[...]]
+      const mappedUsers = teams.map((team: Team) => {
+        return team.members.map((user: User) => ({
+          firstName: user.firstName,
+          lastName: user.lastName,
+        }));
+      });
+      //2 etap -> tworzenie jednej tablicy
+      const cleanUsers: ListUser[] = [];
+      mappedUsers.forEach(team => {
+        cleanUsers.push(...team);
+      });
+      return cleanUsers;
+    };
+
+    const tournamentsResponse = await api.tournament.getAllTournaments(page, 1);
+    this.pagination.rowsNumber = tournamentsResponse.totalRows;
+    this.trueData = tournamentsResponse.tournaments.map((el, index) => {
+      const cleanUsers = mapUsers(tournamentsResponse.tournaments[index].teams);
+      return {
+        id: el.id as string,
+        name: el.name,
+        status: el.isFinished === true ? 'Finished' : 'In progress',
+        participants: cleanUsers,
+      };
+    });
+    console.log(this.trueData);
+  }
```

Comment:
```suggestion
  this.tournaments = await api.tournament.getAllTournaments(page, 1)
```
lol, I think you know what's going on

Reply:
Don't change status/isFinished here for string. Handle it in a template and translate with i18n